components/organisms/o-my-account-placeholder.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <SfTabs :open-tab="1" class="tab-orphan">
```

Comment:
can you change this class into `o-my-account-placeholder`? or add div with class o-my-account-placeholder

Reply:
you are right, added